runners/direct-java/src/main/java/org/apache/beam/runners/direct/BoundedReadEvaluatorFactory.java

```diff
@@ -73,7 +73,7 @@ public void cleanup() {}
    * already done so.
    */
   private <OutputT> TransformEvaluator<?> getTransformEvaluator(
-      final AppliedPTransform<?, PCollection<OutputT>, Bounded<OutputT>> transform) {
```

Comment:
Can we automate the validation that Beam compiles under Eclipse? Without such, this is bound to break again by developers not using Eclipse.